modules/@ergo/core/layouts/default.vue

```diff
@@ -58,8 +58,8 @@ export default {
         ...mapState('authentication', {
             user: state => state.user,
         }),
-        getComponentsForExtended() {
-            return this.$getComponentsForExtended(COMPONENTS.NAVIGATION_BAR);
+        getExtendedComponents() {
```

Comment:
computed property has setter and getter, simplify it by cutting get* - extenededComponents